packages/webdriverio/tests/commands/element/getValue.test.js

```diff
@@ -33,4 +33,18 @@ describe('getValue', () => {
         await elem.getValue()
         expect(request.mock.calls[2][0].uri.path).toBe('/wd/hub/session/foobar-123/element/some-elem-123/attribute/value')
     })
+
+    it('should get value in mobile mode', async () => {
```

Comment:
Does this test fail if you remove your fix for `.getValue()`?

Reply:
yes

Reply:
but how? it does the same assertion as in line 34, no?

Reply:
@christian-bromann because of line 42 :)

Reply:
ahh ok no I see once expand the whole file. Let's do 

```js
if (this.isW3C && !this.isMobile) {
```

then please. I am also wondering if this shouldn't be a bug reported in Appium.

Reply:
Ok, I'll update condition.

It should be either reported to Appium or if it is working as designed we need a link to prove that.
I'll take a look on Monday